### PR TITLE
Scraper admin overview

### PR DESCRIPTION
**Pull request by mcbloch**

### CODE REVIEW

File: server/admin/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                        <a href="{{ scrape_status_result.data_url }}">See data</a>
+                    </div>
+                </div>
+            </div>
+        {% endfor %}
+    </div>
+</div>
+
+<!-- Compiled and minified JavaScript -->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
```
Comment (redfast00) [inline server/admin/templates/index.html:44]: Please also add the SHA256 hash to pin the javascript

**Review by niknetniko — APPROVED** [review]
Pretty good stuff 👍

One question: how is this deployed? As in, where does one go to get a new version deployed?

In the future we might also expose the status of the scrapers as json, that way we can easily add it to sad if we want 😁

**Comment by mcbloch** [comment]
> Pretty good stuff +1
> 
> One question: how is this deployed? As in, where does one go to get a new version deployed?
> 
> In the future we might also expose the status of the scrapers as json, that way we can easily add it to sad if we want grin

the admin folder will be copied in the development folder and will be ran from there. Deploying a version is via the general deploy script.

**Comment by mcbloch** [comment]
Added a symlinked current folder for the current deployment. This way we can do an easy cron job compatible rollback.
This should also fix the issue that docker-compose recognises the current running admin server en updates it instead of creating a new one.

**Comment by mcbloch** [comment]
Related ansible pr for the nginx proxy https://github.com/ZeusWPI/ansible-config/pull/144

**Review by niknetniko — APPROVED** [review]

**Review by niknetniko — APPROVED** [review]

**Pull request merged**